frontend/src/app/models/gyazo.ts

```diff
@@ -0,0 +1,7 @@
+export interface GyazoUpload {
+    image_id: string;
+    permalink_url: string;
```

Comment:
Stick to the single naming convention. I prefer `lowerCamelCase` over `snake_case`